### ISSUE TITLE
update to last version for fix 15.1

### ISSUE THREAD

**Issue by davidmdt21** [gyke69/compiled-zsign#1]
Hello Dear Arian, can you please Update compiled-zsign to last version to fix 15.1 ?
thank you so much 

you know when i use simple code to Compile i get error same below ! but your Compiled File is working fine .

g++ *.cpp common/*.cpp -std=gnu++11 -lcrypto -O3 -o zsign
zsign.cpp: In function ‘int main(int, char**)’:
zsign.cpp:246:10: warning: ignoring return value of ‘int chdir(const char*)’, declared with attribute warn_unused_result [-Wunused-result]
     chdir(szOldFolder);
     ~~~~~^~~~~~~~~~~~~
common/common.cpp: In function ‘bool IsZipFile(const char*)’:
common/common.cpp:278:9: warning: ignoring return value of ‘size_t fread(void*, size_t, size_t, FILE*)’, declared with attribute warn_unused_result [-Wunused-result]
    fread(buf, 1, 2, fp);
    ~~~~~^~~~~~~~~~~~~~~
common/common.cpp: In static member function ‘static void ZLog::Print(int, const char*)’:
common/common.cpp:707:8: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’, declared with attribute warn_unused_result [-Wunused-result]
   write(STDOUT_FILENO, szLog, strlen(szLog));
   ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
common/common.cpp: In static member function ‘static void ZLog::PrintV(int, const char*, ...)’:
common/common.cpp:716:8: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’, declared with attribute warn_unused_result [-Wunused-result]
   write(STDOUT_FILENO, szLog, strlen(szLog));
   ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

**Comment by davidmdt21** [comment]
ok 

> > Hello Dear Arian, can you please Update compiled-zsign to last version to fix 15.1 ? thank you so much
> > you know when i use simple code to Compile i get error same below ! but your Compiled File is working fine .
> > g++ _.cpp common/_.cpp -std=gnu++11 -lcrypto -O3 -o zsign zsign.cpp: In function ‘int main(int, char**)’: zsign.cpp:246:10: warning: ignoring return value of ‘int chdir(const char*)’, declared with attribute warn_unused_result [-Wunused-result] chdir(szOldFolder); ~~~~~^~~~~~~~~~~~~ common/common.cpp: In function ‘bool IsZipFile(const char*)’: common/common.cpp:278:9: warning: ignoring return value of ‘size_t fread(void*, size_t, size_t, FILE*)’, declared with attribute warn_unused_result [-Wunused-result] fread(buf, 1, 2, fp); ~~~~~^~~~~~~~~~~~~~~ common/common.cpp: In static member function ‘static void ZLog::Print(int, const char*)’: common/common.cpp:707:8: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’, declared with attribute warn_unused_result [-Wunused-result] write(STDOUT_FILENO, szLog, strlen(szLog));
> > ```
> > common/common.cpp: In static member function ‘static void ZLog::PrintV(int, const char*, ...)’:
> > common/common.cpp:716:8: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’, declared with attribute warn_unused_result [-Wunused-result]
> > write(STDOUT_FILENO, szLog, strlen(szLog));
> > ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> > ```
> 
> currently zSign doesn't support iOS 15.1, once it supports 15.1 i will compile and upload it for sure.

ok ok  .  thank you so much

**Comment by gyke69** [comment]
> ok
> 
> > > Hello Dear Arian, can you please Update compiled-zsign to last version to fix 15.1 ? thank you so much
> > > you know when i use simple code to Compile i get error same below ! but your Compiled File is working fine .
> > > g++ _.cpp common/_.cpp -std=gnu++11 -lcrypto -O3 -o zsign zsign.cpp: In function ‘int main(int, char**)’: zsign.cpp:246:10: warning: ignoring return value of ‘int chdir(const char*)’, declared with attribute warn_unused_result [-Wunused-result] chdir(szOldFolder); ~~~~~^~~~~~~~~~~~~ common/common.cpp: In function ‘bool IsZipFile(const char*)’: common/common.cpp:278:9: warning: ignoring return value of ‘size_t fread(void*, size_t, size_t, FILE*)’, declared with attribute warn_unused_result [-Wunused-result] fread(buf, 1, 2, fp); ~~~~~^~~~~~~~~~~~~~~ common/common.cpp: In static member function ‘static void ZLog::Print(int, const char*)’: common/common.cpp:707:8: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’, declared with attribute warn_unused_result [-Wunused-result] write(STDOUT_FILENO, szLog, strlen(szLog));
> > > ```
> > > common/common.cpp: In static member function ‘static void ZLog::PrintV(int, const char*, ...)’:
> > > common/common.cpp:716:8: warning: ignoring return value of ‘ssize_t write(int, const void*, size_t)’, declared with attribute warn_unused_result [-Wunused-result]
> > > write(STDOUT_FILENO, szLog, strlen(szLog));
> > > ~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> > > ```
> > 
> > 
> > currently zSign doesn't support iOS 15.1, once it supports 15.1 i will compile and upload it for sure.
> 
> ok ok . thank you so much

added the support for iOS 15 and 15.0.1, zSign has been compiled and added in the release section github already.
